test/lib/sscanf.c

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: GPL-2.0+
```

Comment:
Have you tried compiling and running this?

Reply:
Yes